models/data_ingestion.go

```diff
@@ -197,7 +197,22 @@ const insertSQL = `
     m_docs_scanned_sum,
     m_docs_scanned_min,
     m_docs_scanned_max,
-    m_docs_scanned_p99
+    m_docs_scanned_p99,
+  	m_shared_blks_cnt,
+  	m_shared_blks_hit,
+  	m_shared_blks_read,
+  	m_shared_blks_dirtied,
+  	m_shared_blks_written,
+  	m_local_blks_cnt,
+  	m_local_blks_hit,
+  	m_local_blks_read,
+  	m_local_blks_dirtied,
+  	m_local_blks_written,
+  	m_temp_blks_cnt,
+  	m_temp_blks_read,
+  	m_temp_blks_written,
+  	m_blk_read_time,
+  	m_blk_write_time
```

Comment:
Following the call with @BupycHuk, PostgreSQL metrics should be stored in next way:

```suggestion
  	  	m_shared_blks_hit_cnt,
	  	m_shared_blks_hit_sum,
	  	m_shared_blks_read_cnt,
	  	m_shared_blks_read_sum,
	  	m_shared_blks_dirtied_cnt,
	  	m_shared_blks_dirtied_sum,
	  	m_shared_blks_written_cnt,
	  	m_shared_blks_written_sum,
	  	m_local_blks_hit_cnt,
	  	m_local_blks_hit_sum,
	  	m_local_blks_read_cnt,
	  	m_local_blks_read_sum,
	  	m_local_blks_dirtied_cnt,
	  	m_local_blks_dirtied_sum,
	  	m_local_blks_written_cnt,
	  	m_local_blks_written_sum,
	  	m_temp_blks_read_cnt,
	  	m_temp_blks_read_sum,
	  	m_temp_blks_written_cnt,
	  	m_temp_blks_written_sum,
	  	m_blk_read_time_cnt,
	  	m_blk_read_time_sum,
	  	m_blk_write_time_cnt,
	  	m_blk_write_time_sum
```

Also we need to refactor other related code.